docs/README.md

```diff
@@ -5,10 +5,23 @@ The documentation for this project is built with [Sphinx](https://www.sphinx-doc
 
 ## building the docs locally
 
+Ensure you have installed the `doc-shared` submodule dependencies
+
+```shell
+git submodule update --init
+```
+
+Install Sphinx requirements (note this is `docs/requirements.txt` as opposed to
+the project root's `requirements.txt`)
+
+```shell
+python -m pip install -r requirements.txt
+```
+
 To build the documentation for epx-results locally, execute the following command:
 
 ```terminal
-make html
+make clean html
```

Comment:
`make html` only builds files that have been modified since the last run. `make clean html` rebuilds everthing from source, which is probably safer during development.